client/src/app/home/page.tsx

```diff
@@ -0,0 +1,15 @@
+import { BoardCard } from "@/components/board-card";
+import Link from "next/link";
+
+export default function Home() {
+  return (
+    <div className="p-5">
+      <Link href="/">
```

Comment:
Can we put the Link as a child component of BoardCard and put the href as a prop of BoardCard? This is so that when we map the boards in the home page, we can just pass in the link as a prop of BoardCard